docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -829,7 +825,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.13.0"
+   "version": "3.10.14"
```

Comment:
please use a more recent version. we will not support `python3.10` from next January onward and it's good to live at head (idem for the other notebooks)

Reply:
I just tend to run things with the minimum supported version myself as a sort of safe guard. But I will definitely update then.